devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -101,5 +142,172 @@ protected SpanByte SliceGenericBuffer(int start, int length)
 
             return _genericBuffer;
         }
+
+
+        /// <summary>
+        /// Draws a pixel on the screen.
+        /// </summary>
+        /// <param name="x">The x coordinate on the screen.</param>
+        /// <param name="y">The y coordinate on the screen.</param>
+        /// <param name="useWhite">Indicates if color to be used, default yes unless is inverted screen.</param>
+        public void DrawPixel(int x, int y, bool useWhite = true)
+        {
+            if ((x >= Width) || (y >= Height))
+                return;
```

Comment:
We're using brackets { } around all instructions for if and kind of loops. So please add. This applies in other places. I'll try to point them out and will use the word brackets.